spec.html

```diff
@@ -77,3 +77,19 @@ <h1>String.prototype.replaceAll ( _searchValue_, _replaceValue_ )</h1>
     1. Return _result_.
   </emu-alg>
 </emu-clause>
+
+<emu-clause id="sec-regexp.prototype-@@replaceall">
+  <h1>RegExp.prototype [ @@replaceAll ] ( _string_, _replaceValue_ )</h1>
+  <p>When the `@@replaceAll` method is called with arguments _string_ and _replaceValue_, the following steps are taken:</p>
+  <emu-alg>
+    1. Let _rx_ be the *this* value.
+    1. If Type(_rx_) is not Object, throw a *TypeError* exception.
+    1. Let _flags_ be ? ToString(? Get(_rx_, `"flags"`)).
```

Comment:
if flags is undefined, this should be the empty string, not the string undefined, to match the RegExp constructor.